src/test/java/com/google/jenkins/plugins/k8sengine/KubernetesEngineBuilderTest.java

```diff
@@ -48,121 +54,61 @@
   private static final String TEST_ZONE_B = "us-central1-b";
   private static final String TEST_PROJECT_ID = "test-project-id";
   private static final String OTHER_PROJECT_ID = "other-project-id";
-  private static final String NO_ZONES_PROJECT_ID = "no-zones-project-id";
-  private static final String ERROR_PROJECT_ID = "error-project-id";
   private static final String TEST_CLUSTER = "testCluster";
   private static final String OTHER_CLUSTER = "otherCluster";
   private static final String TEST_CREDENTIALS_ID = "test-credentials-id";
-  private static final String OTHER_CREDENTIALS_ID = "other-credentials-id";
-  private static final String NO_DEFAULT_PROJECT_CREDENTIALS_ID = "empty-project-credentials-id";
-  private static final String PROJECT_ERROR_CREDENTIALS_ID = "project-error-credentials-id";
-  private static final String ERROR_CREDENTIALS_ID = "error-credentials-id";
-  private static final String TEST_ERROR_MESSAGE = "error";
 
   private static Jenkins jenkins;
-  private static KubernetesEngineBuilder.DescriptorImpl descriptor;
+
+  @Mock ClientFactory clientFactory;
+
```

Comment:
No need for extra '\n' in between the instance variables.

Reply:
These instance fields were removed altogether, resolving